src/main/java/org/candlepin/insights/controller/InventoryController.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2009 - 2019 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.insights.controller;
+
+import org.candlepin.insights.inventory.client.InventoryService;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.UUID;
+
+/**
+ * Controller used to interact with the Inventory service.
+ */
+@Component
+public class InventoryController {
+
+    private static Logger log = LoggerFactory.getLogger(InventoryController.class);
+
+    @Autowired
+    private InventoryService inventoryService;
+
+    public InventoryController() {
```

Comment:
No need to specify constructor if it's equivalent to default constructor.

Reply:
DOH! Missed this cleanup.

DONE - Will be included in next commit.